server/manager/src/main/java/org/apache/accumulo/manager/tableOps/split/PreSplit.java

```diff
@@ -63,15 +67,8 @@ public PreSplit(KeyExtent expectedExtent, SortedSet<Text> splits) {
 
   @Override
   public long isReady(FateId fateId, Manager manager) throws Exception {
-
-    // ELASTICITY_TODO intentionally not getting the table lock because not sure if its needed,
-    // revist later when more operations are moved out of tablet server
-
     var opid = TabletOperationId.from(TabletOperationType.SPLITTING, fateId);
 
-    // ELASTICITY_TODO write IT that spins up 100 threads that all try to add a diff split to
-    // the same tablet.
-
     // ELASTICITY_TODO does FATE prioritize running Fate txs that have already started? If not would
```

Comment:
If you look at `FateInterleavingIT.testDefaultInterleaving()` included in #3852, you will notice that I have set the Fate threadpool size to 1 and Fate moves all the transactions forward one step at a time, to include new transactions.